jcache/commons/src/main/java/org/infinispan/jcache/AbstractJCacheNotifier.java

```diff
@@ -37,6 +42,13 @@
 
    private static final boolean isTrace = log.isTraceEnabled();
 
+   private static final ConcurrentLinkedQueue<CountDownLatch> EMPTY_QUEUE = new ConcurrentLinkedQueue<CountDownLatch>() {
```

Comment:
Right, this could be made more pretty. I just need the equivalent on Collections.EMPTY_LIST with minimum amount of code. It's not modified in any way and used just for https://github.com/infinispan/infinispan/pull/3340/files#diff-9377e9175d6d22c89f0cc22f98352feaR113 to make sure I don't lose track and don't "notify" latches which could be added concurrently for the same (cache, key, value) tuple.